time_stamper_ros/cfg/Led.cfg

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
+PACKAGE = "time_stamper_ros"
+
+from dynamic_reconfigure.parameter_generator_catkin import *
+
+gen = ParameterGenerator()
+gen.add("frequency",    int_t,    0, "Clock frequency", 50,  1, 10526315)
+
+size_enum = gen.enum([gen.const("Fps", int_t, 0, "Fps Mode"),
+            gen.const("Exposure", int_t, 1, "Exposure Mode")],
+            "Set Led Mode")
+
+gen.add("mode", int_t, 0, "Led Mode", 1, 0, 3, edit_method=size_enum)
```

Comment:
3 as max value probably makes no sense given that there are only 2 values possible